repoaudit/repoaudit/utils.py

```diff
@@ -1,12 +1,26 @@
+import datetime
 import hashlib
+import json
+import os
```

Comment:
The use of `os` has fallen out of favor with Python 3. Use [pathlib](https://docs.python.org/3/library/pathlib.html) instead.

[Here](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module) is a handy guide.